src/Nethermind.Int256/UInt256.cs

```diff
@@ -689,6 +702,10 @@ private static void Udivrem(ref ulong quot, ref ulong u, int length, in UInt256
                 dLen = 1;
                 shift = LeadingZeros(d.u0);
             }
+            else
+            {
+                throw new DivideByZeroException();
```

Comment:
Is this behaviour change?

Reply:
Looks like in Evm we check for 0  before calling divide, so should be ok (behaviour)

Reply:
Should this be a separate method, non inlinable?

Reply:
Done in [f08fba3](https://github.com/NethermindEth/int256/pull/50/commits/f08fba3694bbc623aaf018b4c3476f817894fa7b)

Reply:
I used the existing method for throwing the exception - just followed the same as `ThrowOverflowException` and am now passing the message as argument